es6.md

```diff
@@ -89,3 +89,52 @@ interface ArrayPattern <: Pattern {
     elements: [ Pattern | null ];
 }
 ```
+
+# Classes
+
+```js
+interface Class <: Node {
+    id: Identifier | null;
+    superClass: Expression;
+    body: ClassBody;
+}
+```
+
+## ClassBody
+
+```js
+interface ClassBody <: Node {
+    type: "ClassBody";
+    body: [ MethodDefinition ];
```

Comment:
Hesitant to rename it considering it's already interoperable and this seems like a minor nit.